decentraland/unity-renderer
# chore: publish the code-coverage in its own step

Fix #842 

## What does this PR change?

The code coverage now is published in another step to let the build finish and publish the artifacts even if the tests fail.

## How to test the changes?

1. Check if the code coverage was upload OK in this PR (the bot will comment the link)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
